apps/src/sites/studio/pages/levels/editors/_all.js

```diff
@@ -3,6 +3,14 @@ import $ from 'jquery';
 
 $(document).ready(initPage);
 
+$("#plusAnswerMap").on("click", () => {
+  $("#plusAnswerMap").prev().clone().insertBefore("#plusAnswerMap");
+});
+
+$("#plusAnswerReference").on("click", () => {
+  $("#plusAnswerReference").prev().clone().insertBefore("#plusAnswerReference");
+});
+
```

Comment:
I don't love adding more jquery to this page, but it might unfortunately be the best option we've got. That said, I would (a) move this code next to other click handlers that we have (lines 53-54), and add a comment for what it's for.